src/transformers/visitors/toContractVisitor.ts

```diff
@@ -19,6 +19,25 @@ const findCustomInputsVisitor = (thisPath: NodePath, thisState: any) => {
   if (thisPath.nodeType !== 'Identifier') return;
   const binding = thisPath.getReferencedBinding(thisPath.node);
   const indicator = thisPath.scope.getReferencedIndicator(thisPath.node, true);
+  if(thisPath.parent.nodeType === 'Return' || thisPath.parentPath.parent.nodeType === 'Return') {
```

Comment:
I think (not sure) this may fail if we have nested nodes (like a mapping with a key which is an array or something) - one soln is to do:
```suggestion
  if(thisPath.getAncestorOfType('Return')) {
```